quiz-bundle-spec.md

```diff
@@ -0,0 +1,71 @@
+# Qwiklabs Quiz Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
+
+## `qwiklabs.yaml` Structure
+
+Here's a sample `qwiklabs.yaml` file with all nested details removed to make it easier to see the general file structure.
+
+```yml
+entity_type: Quiz
+schema_version: 1
+
+# The authoring title of the quiz - not visible to learners.
+title: Super Hard Quiz
+
```

Comment:
Needs a `default_locale`. See https://github.com/CloudVLab/qwiklabs-content-bundle-spec/blob/master/lab-bundle-spec.md#default-locale

No need to explain it more deeply in this PR. Let's note it as a TODO to move the default_locale description in lab bundle to a centralized localization spec file.